core/store/src/main/java/uk/gov/gchq/gaffer/store/serialiser/lengthvalue/ElementIdSerialiser.java

```diff
@@ -103,4 +103,9 @@ public ElementId deserialiseEmpty() throws SerialisationException {
     public boolean preservesObjectOrdering() {
         return false;
```

Comment:
Please could you change this to:
return null != vertexSerialiser && vertexSerialiser.preservesObjectOrdering();